pkg/controllers/restore/controller.go

```diff
@@ -273,13 +275,25 @@ func (h *handler) OnRestoreChange(_ string, restore *v1.Restore) (*v1.Restore, e
 	return restore, err
 }
 
-func (h *handler) restoreCRDs(created map[string]bool, objFromBackupCR ObjectsFromBackupCR) error {
+func (h *handler) restoreCRDs(created map[string]bool, crdWithSubStatus *[]string, objFromBackupCR ObjectsFromBackupCR) error {
 	for crdInfo, crdData := range objFromBackupCR.crdInfoToData {
 		err := h.restoreResource(crdInfo, crdData, false)
 		if err != nil {
 			return fmt.Errorf("restoreCRDs: %v", err)
 		}
 		created[crdInfo.ConfigPath] = true
+
+		specs := crdData.Object[specMapKey].(map[string]interface{})
+		subResources, ok := specs[subResourcesMapKey]
+		if ok {
```

Comment:
If this is not ok or nil, we are fine ignoring it?

Reply:
Right. We are looking for the CRD which has the field `subresources` and add it to the list `crdWithSubStatus`.